.github/workflows/build.yaml

```
@@ -28,24 +28,31 @@ jobs:
         run: |
           source './scripts/ci/lib.sh'
 
-          matrix='{"pre_build_go_binaries": {"name":[]}, "build_and_push_main": {"name":[]} }'
+          matrix='{ "pre_build_go_binaries": { "name":[], "arch":[] }, "build_and_push_main": { "name":[], "arch":[] }, "build_and_push_main_manifests": { "name":[] } }'
```

This job is really just about pushing the multiarch manifests, amiright?
```suggestion
          matrix='{ "pre_build_go_binaries": { "name":[], "arch":[] }, "push_main_multiarch_manifests": { "name":[], "arch":[] }, "build_and_push_main_manifests": { "name":[] } }'
```